docs/getting-started/build.gradle

```diff
@@ -9,8 +9,7 @@ repositories {
 mainClassName = "sample.ElectronicMoneyMain"
 
 dependencies {
-    implementation fileTree(dir: '../../core/build/install/core', include: '*.jar')
-    implementation fileTree(dir: '../../core/build/install/core/lib', include: '*.jar')
+    implementation group: 'com.scalar-labs', name: 'scalardb', version: '3.6.0'
```

Comment:
Added Scalar DB dependency. After this change, we no longer need to build the `scalardb` project.